charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -45,6 +45,11 @@ spec:
         - effect: NoExecute
           operator: Exists
       serviceAccountName: calico-node
+      {{- if semverCompare ">= 1.19" .Capabilities.KubeVersion.GitVersion }}
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      {{- end }}
```

Comment:
Why do you specify a `seccompProfile` for `calico-node` when all of its containers have `privileged: true`?
According to the [documentation](https://kubernetes.io/docs/tutorials/security/seccomp/#before-you-begin) of `seccompProfile`, this has no effect.

Reply:
Yes, this is right. Basically this does nothing with the current setup of `calico-node` because the single container runs as privileged. However it acts as guardrails against:
1. adding a new container to the pod without seccomp profile
2. removing the `privileged: true` from the container and still running as `Unconfined`

Reply:
Do you plan on proposing those changes in the upstream calico repository as well? It feels shaky if we add it here without having upstream commitment.

Reply:
You are right. I will check if they have any ongoing (or planned) works in this direction and if not I will propose the changes I made here.